README.md

```diff
@@ -247,6 +247,7 @@ examples = null
 | Argument                  | Type                                    | Default      | Description                                                                                                                                  |
 | ------------------------- | --------------------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
 | `labels`                  | `str`                                   |              | Comma-separated list of labels.                                                                                                              |
+| `label_descriptions`      | `Optional[Dict[str, str]]`              | `None`       | Optional dict of label: description of label to better instruct the LLM on what to extract for each label.                                                                                                              |
```

Comment:
(plus reformatting)

```suggestion
| `label_descriptions`      | `Optional[Dict[str, str]]`              | `None`       | Optional dict of label descriptions: descriptions of labels to better instruct the LLM on what to extract for each label.                                                                                                              |
```

Reply:
Hmm I think the way it is now makes more sense? 

The dict will look like:

```
{
    "PERSON": "Any named individual in the text, e.g. ..."
}
```

So spelling out that it's a dict of `label: description of label` is correct

Reply:
That is not clear at all from the current formatting.

Reply:
Ok, I just rewrote this. How's the new definition?